explainer.md

```diff
@@ -67,6 +72,22 @@ interface PictureInPictureWindow {
 
   attribute EventHandler onresize;
 };
+
+interface PictureInPictureControl {
+  attribute DOMString id;
+  attribute DOMString label; // Description of the action.
+  attribute FrozenArray<MediaImage> icons;
+};
+
+dictionary MediaImage {
+  required USVString src;
+  DOMString sizes = "";
+  DOMString type = "";
+};
+
+interface PictureInPictureControlsEventHandler : EventHandler {
```

Comment:
Please add `[Constructor(DOMString type, PictureInPictureControlEventInit eventInitDict)]` and 

```
dictionary PictureInPictureControlEventInit : EventInit {
    required DOMString id;
};
```